doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/HStackExamples.swift

```diff
@@ -0,0 +1,227 @@
+    var body: some View {
+        NavigationStack {
+            HStack(alignment:.firstTextBaseline) {
+                Text("First").font(.largeTitle)
+                Text("Second").font(.title)
+                Text("Third").font(.title2)
+            }.border(Color.blue, width: 1)
+        }.navigationTitle("Content First Text Baseline Aligned")
+    }
+}
+
+
+struct HStackContentLastTextBaselineAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack(alignment:.lastTextBaseline) {
+                Text("First").font(.largeTitle)
+                Text("Second").font(.title)
+                Text("Third").font(.title2)
+            }.border(Color.blue, width: 1)
+        }.navigationTitle("Content Last Text Baseline Aligned")
+    }
+}
+
+struct HStackExceedsAvailableSpacePage : View {
+    var body: some View {
+        NavigationStack {
+            HStack() {
+                Rectangle()
+                    .foregroundColor(Color.red)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.green)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.blue)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.purple)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.orange)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.cyan)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.yellow)
+                    .frame(width: 100)
+            }
+        }.navigationTitle("HStack Exceeds Space")
+    }
+}
+
+
+struct LazyHStackExceedsAvailableSpacePage : View {
+    var body: some View {
+        NavigationStack {
+            LazyHStack() {
+                Rectangle()
+                    .foregroundColor(Color.red)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.green)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.blue)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.purple)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.orange)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.cyan)
+                    .frame(width: 100)
+                Rectangle()
+                    .foregroundColor(Color.yellow)
+                    .frame(width: 100)
+            }
+        }.navigationTitle("Lazy HStack Exceeds Space")
+    }
+}
+
+
+
+struct HStackLeftAlignedPage_Previews: PreviewProvider {
```

Comment:
Let's place each of these preview configurations immediately below the view they apply to.

Reply:
Done! Added previews just below the layouts.